w3m-form.el

```diff
@@ -439,7 +440,7 @@ fid=\\([^/]+\\)/type=\\([^/]+\\)/name=\\([^/]*\\)/id=\\(.*\\)$"
 		    (let ((hseq (nth 2 (w3m-action (point))))
 			  (value (w3m-form-get form id)))
 		      (when (> hseq 0)
-			(setq textareas (cons (cons hseq value) textareas)))
+			(push textareas (cons hseq value) textareas))
```

Comment:
I think this change contains a mistake.  Do you mean (push (cons hseq value) textareas) ?

Reply:
Yes! Thanks. Fixed.